packages/driver/cypress/integration/e2e/multi-domain/multi_domain_config_env_spec.ts

```diff
@@ -0,0 +1,263 @@
+['config', 'env'].forEach((fnName) => {
+  // @ts-ignore
+  describe(`multi-domain - Cypress.${fnName}()`, { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
```

Comment:
Should the `config` and `env` be reset in an `after` hook?

Reply:
I don't think so since 1-2 of the tests have interdependencies. It's not the best practice, but we need a way to show that the config is resynced to `switchToDomain` without explicitly setting config values in the primary in the test. We could do this in a single test with multiple `switchToDomain` calls, which could work since all these callbacks run synchronous. We shouldn't even need to run the visit in those cases. What do you think?

Reply:
Since it's a `SetterGetter` I think we can just call the `reset` method

Reply:
fixed in ede74adaf2